recipes/diligent-core/all/conanfile.py

```diff
@@ -146,12 +148,21 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "Licenses"))
         self.copy("License.txt", dst="licenses", src=self._source_subfolder)
 
-    def package_info(self):
-        if self.settings.build_type == "Debug":
-            self.cpp_info.libdirs.append("lib/source_subfolder/Debug")
-        if self.settings.build_type == "Release":
-            self.cpp_info.libdirs.append("lib/source_subfolder/Release")
+        bin_path = os.path.join(self.package_folder, "lib", "source_subfolder", str(self.settings.build_type))
+
+        for dll in glob.glob(os.path.join(self.package_folder, bin_path, "*.dll")):
```

Comment:
Now that you're in package(self), you can use self.copy, and remove the glob and shutil import